### DISCUSSION TITLE
Cargo spawn points 588 and 590 in PVP_RaateColony have their tags missing

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#14942] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/a7cb8bf8-5ae3-4c54-8189-714d34da51e9)
![image](https://github.com/user-attachments/assets/43f5ad8b-e7cc-479e-ab25-a6c115fcb7d7)

### Reproduction steps

- open submarine editor
- open PVP_RatteColony
- check the spawnpoints in the middle

### Bug prevalence

Just once

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/694cc2c9ebba4d548cb662a5d0caa5b3b5e99c4b (commit in our private repo), fix coming up in a hotfix soon.
